src/App.tsx

```diff
@@ -12,18 +12,13 @@ export const App: React.FC<{}> = () => (
   <StylesProvider injectFirst>
     <MuiThemeProvider theme={theme}>
       <ThemeProvider theme={theme}>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            {ITProfessions.map(profession => (
-              <Route path={`/details/${profession.route}`} key={profession.id}>
-                <DetailPage profession={profession} />
-              </Route>
-            ))}
-          </Switch>
-        </Router>
+        <Switch>
+          <Route exact path="/">
+            <HomePage />
+          </Route>
+          <Redirect exact path="/details" to={`/details/${ITProfessions[0].route}`} />
```

Comment:
This redirect will navigate to first profession url if the user tries to enter `details` route without any profession id :)